frappe/public/js/frappe/model/model.js

```diff
@@ -650,6 +650,27 @@ $.extend(frappe.model, {
 		}
 	},
 
+	round_floats_childs_in: function(arr, fieldnames) {
+		var cached_precision = [];
+		$.each(arr || [], function(k, item) {
```

Comment:
use ES6 instead of jquery? `arr.forEach` ?

Reply:
done